subliminal/providers/subs4series.py

```diff
@@ -129,9 +129,9 @@ def _get_suggestions(self, title):
 
         """
         # make the search
-        logger.info('Searching show ids with %r', title)
-        r = self.session.get(self.server_url + self.search_url % title, headers={'Referer': self.server_url},
-                             timeout=10)
+        logger.info('Searching show ids with {!r}'.format(title))
```

Comment:
For logging, you should just follow the standard logging in python. Do not apply the string formatting. It was correct how it was before